routes/notifications.php

```diff
@@ -1,13 +1,6 @@
 <?php
 
 // Notifications
-$router->group([
+resource($router, 'notifications', 'NotificationsController', [
     'middleware' => ['auth:api', 'scope:notifications'],
```

Comment:
We could almost include scope middleware automatically too, with an optional override. Same with auth.
It depends how much we want to match the original laravel API